src/extension.ts

```diff
@@ -78,6 +66,31 @@ function useStylefmt(document: vscode.TextDocument, range: vscode.Range): Promis
 		}));
 }
 
+/**
+ * Resolve Stylefmt config
+ */
+function useStylefmt(document: vscode.TextDocument, range: vscode.Range): Promise<IResult> {
+	const settings: IStylefmtOptions = vscode.workspace.getConfiguration().get('stylefmt');
+	const configResolver = new ConfigResolver(vscode.workspace.rootPath);
+	const options = {
+		packageProp: 'stylelint',
+		configFiles: [
+			'.stylelintrc',
+			'stylelint.config.js'
+		],
+		editorSettings: settings.config
+	};
+
+	return configResolver.scan(document.uri.fsPath, options).then((resolved) => {
+		return stylefmtProcess(document, range, {
+			configBasedir: settings.configBasedir || vscode.workspace.rootPath,
+			config: resolved.json
+		});
+	}).catch(() => {
+		return stylefmtProcess(document, range);
```

Comment:
But stylefmt supports usage without config which indicates to me that that is the appropriate behaviour. However it is up to you, else I believe devs should disable the plugin for workspaces where it is not needed.